src/status_im/ui/screens/home/views.cljs

```diff
@@ -148,46 +148,47 @@
 (defn render-fn [home-item]
   [inner-item/home-list-item home-item])
 
-(defn communities-and-chats [chats status-community loading? search-filter hide-home-tooltip?]
+(defn communities-and-chats [chats communities loading? search-filter hide-home-tooltip?]
   (if loading?
     [react/view {:flex 1 :align-items :center :justify-content :center}
      [react/activity-indicator {:animating true}]]
     (if (and (empty? chats)
-             (not status-community)
+             (empty? communities)
              (empty? search-filter)
              hide-home-tooltip?
              (not @search-active?))
       [welcome-blank-page]
-      [list/flat-list
-       {:key-fn                       :chat-id
-        :keyboard-should-persist-taps :always
-        :data                         chats
-        :render-fn                    render-fn
-        :header                       [:<>
-                                       (when (or (seq chats) @search-active? (seq search-filter))
-                                         [search-input-wrapper search-filter chats])
-                                       [referral-item/list-item]
-                                       (when (and (empty? chats)
-                                                  (not status-community)
-                                                  (or @search-active? (seq search-filter)))
-                                         [start-suggestion search-filter])
-                                       (when status-community
-                                         ;; We only support one community now, Status
-                                         [communities.views/status-community status-community])
-                                       (when (and status-community
-                                                  (seq chats))
-                                         [quo/separator])]
-        :footer                       (if (and (not hide-home-tooltip?) (not @search-active?))
-                                        [home-tooltip-view]
-                                        [react/view {:height 68}])}])))
+      [react/view
```

Comment:
revert during resolving conflict, this form should be in flatlist header

Reply:
there are two flat lists now, one for communities and one for chats, and a single header, the flat list for communities is not shown if no communities are available, and similarly for chats. 
Should I still put it in the flatlist? (the logic would be a bit awkward i.e if both, put in the first flat list, if only chats puts in the chat flatlist).
Otherwise you could have a single flatlist managing both chats and communities, but not sure if that's a good idea, as we also need some separator etc.
Or I misunderstand your comment?

Reply:
so we had scroll view before and we had search bar and flatlist of chats inside, we should never have nested lists, so i moved searchbar into flatlist header and removed scroll list,  we should defenetly have communities and chats in the same flatlist

Reply:
I have joined the two lists now and they shop a separator on the last item